presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorDistributedQueries.java

```diff
@@ -74,12 +76,22 @@ public void testShardUuidHiddenColumn()
         MaterializedResult actualResults = computeActual("SELECT *, \"$shard_uuid\" FROM test_shard_uuid");
         assertEquals(actualResults.getTypes(), ImmutableList.of(DATE, BIGINT, VARCHAR));
         List<MaterializedRow> actualRows = actualResults.getMaterializedRows();
+        String arbitraryUuid = null;
         for (MaterializedRow row : actualRows) {
             Object uuid = row.getField(2);
             assertInstanceOf(uuid, String.class);
             // check that the string can be parsed into a UUID
             UUID.fromString((String) uuid);
+            arbitraryUuid = (String) uuid;
         }
+        assertNotNull(arbitraryUuid);
```

Comment:
This is just asserting we had a row. If it was null or invalid, the loop would throw.

So before the loop, assert actualRows is not empty.